src/commands/flow/internal/spec/checkPredecessorDeadlock.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { checkPredecessorDeadlock } from '..'
+
+describe('checkPredecessorDeadlock', () => {
+  it('should return true, if predecessorDeadlock is present and itself', () => {
+    const flows = {
+      flow1: {
+        jobs: [{ location: 'job' }],
+        predecessors: ['flow1']
+      }
+    }
+
+    expect(checkPredecessorDeadlock(flows)).toEqual(['flow1', 'flow1'])
+  })
+
+  it('should return true, if predecessorDeadlock is present and direct', () => {
```

Comment:
Please improve the test case description

Reply:
fixed